site/sidebar.js

```diff
@@ -137,6 +137,9 @@ module.exports = {
             "voice/webhooks/mediaStreamStarted",
             "voice/webhooks/mediaStreamRejected",
             "voice/webhooks/mediaStreamStopped",
+            "voice/webhooks/realTimeTranscriptionStarted",
+            "voice/webhooks/realTimeTranscriptionRejected",
+            "voice/webhooks/realTimeTranscriptionStopped",
```

Comment:
oh crud, do we need to add `trancriptionAvailable` to this?

Reply:
Yeah we do need to

Reply:
Added